docs/root/configuration/overview/v2_overview.rst

```diff
@@ -0,0 +1,378 @@
+    address:
+      socket_address: { address: 127.0.0.1, port_value: 9901 }
+  
+  dynamic_resources:
+    lds_config:
+      api_config_source:
+        api_type: GRPC
+        cluster_name: [xds_cluster]
+    cds_config:
+      api_config_source:
+        api_type: GRPC
+        cluster_name: [xds_cluster]
+
+  static_resources:
+    clusters:
+    - name: xds_cluster
+      connect_timeout: 0.25s
+      type: STATIC
+      lb_policy: ROUND_ROBIN
+      hosts: [{ socket_address: { address: 127.0.0.3, port_value: 5678 }}]
+
+The management server could respond to LDS requests with:
+
+.. code-block:: yaml
+
+  version_info: "0"
+  resources:
+  - "@type": type.googleapis.com/envoy.api.v2.Listener
+    name: listener_0
+    address:
+      socket_address:
+        address: 127.0.0.1
+        port_value: 10000
+    filter_chains:
+    - filters:
+      - name: envoy.http_connection_manager
+        config:
+          stat_prefix: ingress_http
+          codec_type: AUTO
+          rds:
+            route_config_name: local_route
+            config_source:
+              api_config_source:
+                api_type: GRPC
+                cluster_name: [xds_cluster]
+          http_filters:
+          - name: envoy.router
+
+The management server could respond to RDS requests with:
+
+.. code-block:: yaml
+
+  version_info: "0"
+  resources:
+  - "@type": type.googleapis.com/envoy.api.v2.RouteConfiguration
+    name: local_route
+    virtual_hosts:
+    - name: local_service
+      domains: ["*"]
+      routes:
+      - match: { prefix: "/" }
+        route: { cluster: some_service }
+
+The management server could respond to CDS requests with:
+
+.. code-block:: yaml
+
+  version_info: "0"
+  resources:
+  - "@type": type.googleapis.com/envoy.api.v2.Cluster
+    name: some_service
+    connect_timeout: 0.25s
+    lb_policy: ROUND_ROBIN
+    type: EDS
+    eds_cluster_config:
+      eds_config:
+        api_config_source:
+          api_type: GRPC
+          cluster_name: [xds_cluster]
+
+The management server could respond to EDS requests with:
+
+.. code-block:: yaml
+
+  version_info: "0"
+  resources:
+  - "@type": type.googleapis.com/envoy.api.v2.ClusterLoadAssignment
+    cluster_name: some_service
+    endpoints:
+    - lb_endpoints:
+      - endpoint:
+          address:
+            socket_address:
+              address: 127.0.0.2
+              port_value: 1234
+
+Management server
+-----------------
+
+A v2 xDS management server will implement the below endpoints as required for
+gRPC and/or REST serving.  In both streaming gRPC and
+REST-JSON cases, a :ref:`DiscoveryRequest <envoy_api_msg_DiscoveryRequest>` is sent and a
+:ref:`DiscoveryResponse <envoy_api_msg_DiscoveryResponse>` received following the
+`xDS protocol <https://github.com/envoyproxy/data-plane-api/blob/master/XDS_PROTOCOL.md>`_.
+
+gRPC streaming endpoints
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. http:post:: /envoy.api.v2.ClusterDiscoveryService/StreamClusters
```

Comment:
I think the small thing that is missing here is what do I configure on the Envoy side to engage these APIs. Could we add a very small snippet below each one that 1) links to the config source that needs to be configured, 2) says which config type to use? I think if we had that it would be totally obvious and tie it all together.